FAIL
Kernel panic in run: PANIC at ../../threads/palloc.c:314 in palloc_free_multiple(): executed an unreachable statement
Call stack: 0x800421894c 0x800420c013 0x800420c109 0x800420d400 0x800420d483 0x800420d506 0x800420d5cf 0x800421c474 0x800421c410 0x800420717f 0x800421db73 0x800421d8be 0x800421d771
Translation of call stack:
0x000000800421894c: debug_panic (lib/kernel/debug.c:32)
0x000000800420c013: palloc_free_multiple (threads/palloc.c:316)
0x000000800420c109: palloc_free_page (threads/palloc.c:329)
0x000000800420d400: pt_destroy (threads/mmu.c:157 (discriminator 2))
0x000000800420d483: pgdir_destroy (threads/mmu.c:167 (discriminator 2))
0x000000800420d506: pdpe_destroy (threads/mmu.c:177 (discriminator 2))
0x000000800420d5cf: pml4_destroy (threads/mmu.c:196)
0x000000800421c474: process_cleanup (userprog/process.c:302)
0x000000800421c410: process_exit (userprog/process.c:275)
0x000000800420717f: thread_exit (threads/thread.c:331)
0x000000800421db73: remove (userprog/syscall.c:242)
0x000000800421d8be: syscall_handler (userprog/syscall.c:130)
0x000000800421d771: no_sti (userprog/syscall-entry.o:?)
